### DISCUSSION TITLE
Star Wars Detours

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#520] [Curiosidades]
Post de referência; https://www.vish.blog.br/curiosidades/entretenimento/star-wars-detours/

**Comment by brunoaugusto** [comment]
Eu PRECISO assistir essa série!

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/curiosidades/entretenimento/star-wars-detours/#comment_1087537273>):

> Eu PRECISO assistir essa série!

Meu cachorro me disse que o primeiro episódio vazou
